x/concentrated-liquidity/types/codec.go

```diff
@@ -10,8 +10,8 @@ import (
 
 func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterInterface((*ConcentratedPoolExtension)(nil), nil)
-	cdc.RegisterConcrete(&MsgCreatePosition{}, "osmosis/cl-create-position", nil)
-	cdc.RegisterConcrete(&MsgWithdrawPosition{}, "osmosis/cl-withdraw-position", nil)
+	cdc.RegisterConcrete(&MsgCreatePosition{}, "osmosis/concentratedliquidity/create-position", nil)
```

Comment:
Can we make the amino codec as shortest as possible? Afaik we still have the word limit issue on ledger for amino codec msgs

Reply:
Was not aware of this, will change it, thanks!